 <div style="display:inline-block; position:absolute; top:0; right:0; color: #230e3e; font:italic 1em serif;">$\LaTeX$<span style="display:inline-block; vertical-align:bottom">
    <a href="https://www.mathjax.org">
    <img style="height:1.5em" title="Powered by MathJax"
    src="https://www.mathjax.org/badge/mj_logo.png"
    border="0" alt="Powered by MathJax" >
</a></span></div>

<div style="display:inline-block; position:absolute; top:0; left:0; color: #230e3e; font:italic 1em serif;"><a href="https://www.julialang.org">
    <img style="height:2.25em; vertical-align:top" title="Powered by Julia"
    src="./figures/julia-logo-mask.png"
    border="0" alt="Powered by Julia" >
</a></div>
</a></div>$\require{cancel} \require{mhchem} \require{bbox} \require{color} \require{enclose} \require{AMSmath} \require{AMSsymbols}$

<header style="background-color:#e7e2ee; color:#461d7c; line-height:2em; padding-left:1em; padding-right:1em; bottom:0; float:bottom; text-align:right; float:right; font-family:serif; font-size:small;">Çengel, Y. A., and Boles, M. A. (2001). <i>Thermodynamics: An engineering approach.</i> Boston: McGraw-Hill.
</header>

**5.62.** Refrigerant-134a is throttled from the saturated liquid state at 700 kPa to a pressure of 160 kPa. Determine the temperature drop during this process and the final specific volume of the refrigerant. 

![Cengelboles7ed4-126](./figures/ch05_cen98179_p05062_lg.jpg)


### Given
<ol> 
$ \text{R-134a}\left|\small \begin{array}{l|l} P_1 = 700\, \ce{kPa} & P_2 = 160\,\ce{kPa} \\[-3pt]
    x_1:\text{sat liquid} \end{array} \right. \\[12pt]$     
 

### Find
<ol type="A">  
    
  1. $T_2$     
  2. $v_2$ 


### Get Organized   

The most important thing to know about throttling processes is that they are **isenthalpic.** Beyond that, they are steady state devices.   

### Attack 

 Today, we'll use Julia, with _Unitful.jl_, a package for calculation with physical units.

In [1]:
using Unitful;       # Unitful units package

Resolve state 1. Use $P_2$ and $h_2 = h_1$ to resolve state 2. Get to work.   

  $\quad \text{R-134a} \left| \begin{array}{ll} 
         \scriptsize \begin{array}{l}{\textit{700 kPa}}\\[-12pt] {\textit{sat liquid}}\end{array} & \small h_1 = 86.78\, \ce{kJ/kg} & \small T_1 = T_{sat@700kPa} = 26.72\, \ce{°C}   \\ \hline
         \scriptsize \begin{array}{l}{\textit{160 kPa}}\\[-12pt] {h_2 = h_1}\end{array} & \small h_{f@160kPa} = 29.78\, \ce{kJ/kg} & \small h_{g@160kPa} = 237.97\ \ce{kJ/kg} & \small v_{f@160kPa} = 0.7435\ \ce{cm^3/g} & \small v_{g@160kPa} = 0.1229\ \ce{m^3/kg} 
         \end{array}\right. \\[12pt]$    

Based on $h_{2@160kPa},$ we have saturation conditions, so 

$$T_2 = T_{sat@160kPa} = -15.62\, \ce{°C},$$   

and  

$$\ \displaystyle \frac{v_2 - v_f}{v_g - v_f} = x_2 = \frac{h_2 - h_f}{h_g - h_f}.\\[12pt]$$   


Solve for $v_2,$ put your feet up, and enjoy a beverage.

In [2]:
h₁ = 86.78u"kJ/kg"
h₂ = h₁
h𝑓 = 29.78u"kJ/kg"
h𝑔 = 237.97u"kJ/kg"

v𝑓 = 0.7435e-3u"m^3/kg"
v𝑔 = 0.1229u"m^3/kg"

x₂ = (h₂ - h𝑓)/(h𝑔 - h𝑓)

v₂ = v𝑓 + x₂ * (v𝑔 - v𝑓)

0.03418852857966281 m^3 kg^-1

<p style = "width:100vw; background-image:  url(./figures/bacgroundTiles.png); line-height: 0.35em" >&nbsp;</p>

In [3]:
T₁ = 26.72u"°C"
T₂ = -15.62u"°C"

ΔT = T₂ - T₁

-42.339999999999996 K

<div class="panel-success"><div class="panel-heading"><div class="badge">A</div>
$\ \normalsize \Delta T = - 42.34\, \ce{°C}\qquad$ <div class="badge">B</div> $\ \normalsize v_2 = 0.0342\, \ce{m^3/kg}$
</div></div>

<footer style="background-color: #461d7c; color:rgba(253,208,35,0.35); line-height:2em; padding-left:1em; padding-right:0.5em; bottom:0; float:bottom; text-align:right;"><i>ψᵟ</i></footer>